src/cms/src/api/emergency-alert/content-types/emergency-alert/lifecycles.js

```diff
@@ -1,23 +1,32 @@
 "use strict";
 
+const format = require('date-fns/format')
+const utcToZonedTime = require('date-fns-tz/utcToZonedTime')
+
+const formatDateToPacificTime = (dateString) => {
+  // Strapi returns the date in ISO format e.g. 2025-01-01T00:00:00.000Z
+  // Convert dateString to Pacific Time
+  const pacificTime = utcToZonedTime(dateString, 'America/Los_Angeles')
+  // Format the date in YYYY-MM-DD e.g. 2025-01-01
+  return format(pacificTime, 'yyyy-MM-dd')
```

Comment:
I haven't used `date-fns-tz` before but it looks like the function we really want is `formatInTimezone` which would let us do this all on one line: https://github.com/marnusw/date-fns-tz?tab=readme-ov-file#formatintimezone

Reply:
Thanks! Tried `formatInTimezone` but was getting an error: `Format string contains an unescaped latin alphabet character 'A'.` for some reason ...